src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,71 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  setSortFilter: (filter: string) => void;
+  sortFilter: string;
+  setSortText: (text:string) => void;
+  sortText: string;
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+};
+
+export const TodoFilter:React.FC<Props> = ({
+  setSortFilter,
+  sortFilter,
+  setSortText,
+  sortText,
+}) => {
+  const handleoChangeValue
+    = (event: React.ChangeEvent<HTMLSelectElement>) => {
+      setSortFilter(event.currentTarget.value);
+    };
+
+  const handleChangeText
+    = (event: React.ChangeEvent<HTMLInputElement>) => {
+      setSortText(event.currentTarget.value);
+    };
```

Comment:
I don't sure about it, but it's bad idea to change props in child component

Reply:
I did changes onClick. I also don't sure. But it works

Reply:
@Jamal-Amoian @NataliyaAn 

it's ok. but describe all chenging logic in parent and pass just a function which you will use onClick